src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.cs

```diff
@@ -71,7 +90,7 @@ private async Task LoadNativeSplashScreen()
 	}
 
 
-#if !__ANDROID__ && !__IOS__ && !(WINDOWS || WINDOWS_UWP)
+#if !__ANDROID__ && !__IOS__ && !(WINDOWS || WINDOWS_UWP || NETSTANDARD2_0)
```

Comment:
I think it's clearer that way.

```suggestion
#if !__ANDROID__ && !__IOS__ && !(WINDOWS || WINDOWS_UWP) && !NETSTANDARD2_0
```

Though is this ever true now?

Reply:
yeh, there's still macos/catalyst